### PR TITLE
Introduction of ggpubr

### PR DESCRIPTION
**Pull request by yhlpkucu**
It's a brief introduction of ggpubr package.

### CODE REVIEW

File: Brief_Introduction_of_ggpubr.Rmd [hunk 1]
```diff
@@ -0,0 +1,282 @@
+#Brief Introduction of ggpubr
```
Comment (alexandriaGuo) [inline Brief_Introduction_of_ggpubr.Rmd:1]: please add a space after #

**Comment by alexandriaGuo** [comment]
please rename the .Rmd file so that it does not use any capital letters

File: Brief_Introduction_of_ggpubr.Rmd [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+
+Hanlin Yan
+
+### 1. Introduction
```
Comment (alexandriaGuo) [inline Brief_Introduction_of_ggpubr.Rmd:5]: please remove the numbering in all headings/subheadings in the .Rmd file

File: Brief_Introduction_of_ggpubr.Rmd (hunk 1 above)
Comment (yhlpkucu) [reply Brief_Introduction_of_ggpubr.Rmd]: Fixed

File: Brief_Introduction_of_ggpubr.Rmd (hunk 2 above)
Comment (yhlpkucu) [reply Brief_Introduction_of_ggpubr.Rmd]: fixed

**Pull request merged**